src/main/java/org/thoughtcrime/securesms/accounts/AccountSelectionListFragment.java

```diff
@@ -113,9 +114,35 @@ private void onContextItemSelected(MenuItem item, int accountId) {
     case R.id.menu_mute_notifications:
       onToggleMute(accountId);
       break;
+    case R.id.menu_set_tag:
+      onSetTag(accountId);
+      break;
     }
   }
 
+  private void onSetTag(int accountId) {
+    Activity activity = getActivity();
+    if (activity == null) return;
+    AccountSelectionListFragment.this.dismiss();
+
+    DcContext dcContext = DcHelper.getAccounts(activity).getAccount(accountId);
+    View view = View.inflate(activity, R.layout.single_line_input, null);
+    EditText inputField = view.findViewById(R.id.input_field);
+    inputField.setHint(R.string.tag_field_hint);
```

Comment:
i thought somehow, setting placeholders is also possible directly in`.xml`, but maybe i'm wrong :)

Reply:
the `single_line_input` layout is used for set contact profile name and add proxy dialogs, it is generic for any dialog wanting to add a single line input field, so the hint can't be set in the xml